src/qt/transactionrecord.h

```diff
@@ -40,19 +34,19 @@ class TransactionStatus
     };
 
     /// Transaction counts towards available balance
-    bool countsForBalance;
+    bool countsForBalance{false};
     /// Sorting key based on status
     std::string sortKey;
 
     /** @name Generated (mined) transactions
        @{*/
-    int matures_in;
+    int matures_in{0};
     /**@}*/
 
     /** @name Reported status
        @{*/
-    Status status;
-    qint64 depth;
+    Status status{Unconfirmed};
+    qint64 depth{0};
     qint64 open_for; /**< Timestamp if status==OpenUntilDate, otherwise number
```

Comment:
b279331569e42cab721949d6669a452d6ba1558a

nit, initialize `open_for` or move this commit after 064bbdf243?

Reply:
Done. Thanks.